WebDriverAgentLib/Routing/FBSession.m

```diff
@@ -53,6 +54,22 @@ @implementation FBSession (FBAlertsMonitorDelegate)
 
 - (void)didDetectAlert:(FBAlert *)alert
 {
+  NSString *autoClickAlertSelector = FBConfiguration.autoClickAlertSelector;
+  if ([autoClickAlertSelector length] > 0) {
+    @try {
+      NSArray<XCUIElement*> *matches = [alert.alertElement fb_descendantsMatchingClassChain:autoClickAlertSelector
+                                                                shouldReturnAfterFirstMatch:YES];
+      if (matches.count > 0) {
+          [[matches objectAtIndex:0] tap];
+      }
```

Comment:
Not a strong opinion but would it be helpful to leave log if the element has over 2? So something like `<given selector> matches over two elements. clicking the first one` for future investigation

Reply:
I would like to keep the current behaviour for now as multiple elements matching might be less performant than single element match, and I prefer performance in this scenario